resources/function_help/json/array_foreach

```diff
@@ -8,14 +8,17 @@
     "description": "an array"
   }, {
     "arg": "expression",
-    "description": "an expression to evaluate on each item. The variable `@element` will be replaced by the current value."
+    "description": "an expression to evaluate on each item. The variable `@element` will be replaced by the current value and the variable `@counter` by the current counter (starting with 0)."
```

Comment:
Would `index` be clearer/usual than `counter`, at least in the description?

Reply:
Both is used, e.g. from https://en.wikipedia.org/wiki/For_loop#Loop_counters

> Loop counters change with each iteration of a loop, providing a unique value for each individual iteration.

But I do not have a fixed opinion, I am happy to change it if desired.

I guess the following is what you've suggested?

```suggestion
    "description": "an expression to evaluate on each item. The variable `@element` will be replaced by the current value and the variable `@counter` by the current index (starting with 0)."
```